### PR TITLE
Styling flight show page

### PR DESCRIPTION
**Pull request by GeorgieGirl24**
- [x] Updated Files:
  * gem file
  * gem lock
  * application.html.erb
  * assests/images
  * application.js
  * config/application.rb
  * application.scss
- [ ] Repo Representation: Front End, Back End, Amedaus, Open Weather, Google 
 * Front End

- [ ] Updated README.md with added information

- [ ] Collabotors:
 * Connor Ferguson
 * Sheryl Stillman
 * Nick King
 * George Soderholm

### CODE REVIEW

File: app/controllers/flight_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class FlightController < ApplicationController
```
Comment (stillsheryl) [inline app/controllers/flight_controller.rb:1]: Is this supposed to be an addition to our flights controller?

File: app/views/flight/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1>Your results</h1>
```
Comment (stillsheryl) [inline app/views/flight/show.html.erb:1]: I think this file and the controller might be old ones that we had already updated to be plural...

**Review by dunlapww — COMMENTED** [review]
code looks good, looking forward to seeing it! 
One, thing - Todd and I pushed up a PR a bit ago and have the weather images in a 'weather_images' folder within images.  Maybe could you move your images to that path?

**Review by SageOfCode — APPROVED** [review]
Awesome, I can't wait to see what the new font looks like.

File: spec/features/welcome/landing_spec.rb [hunk 3]
```diff
@@ -23,50 +23,41 @@
...
-    within('.navbar-header') do
-      expect(page).not_to have_link('Logout')
+      within('.navbar-header') do
+        click_link 'Logout'
+      end
+      expect(page).not_to have_button('Search Locations')
+      within('.navbar-header') do
+        expect(page).not_to have_link('Logout')
+      end
+      expect(page).to have_content('You are now logged out.')
```
Comment (JesseMellinger) [inline spec/features/welcome/landing_spec.rb:57]: Test looks good!

File: spec/features/welcome/landing_spec.rb [hunk 4]
```diff
@@ -23,50 +23,41 @@
...
-      click_link 'Log In with Google'
+      stub_omniauth
+      visit root_path
+      expect(page).to_not have_button('Search Locations')
+      expect(page).to have_link('Log In with Google')
+      within('.login-link') do
+        click_link 'Log In with Google'
+      end
+      expect(page).to have_button('Search Locations')
+      expect(page).to have_content('Welcome, William!')
```
Comment (JesseMellinger) [inline spec/features/welcome/landing_spec.rb:37]: Personalized welcome message. Nice.

**Review by JesseMellinger — APPROVED** [review]

**Comment by cpfergus1** [comment]
Waited 15 min for a PR sorry for the team merge

**Pull request merged**